src/lib/yang/binary.lua

```diff
@@ -210,11 +210,19 @@ local function data_emitter(production)
          end
       end
    end
+   local native_types = {
+      ["instance-identifier"] = true,
+      empty = true,
+      enumeration = true,
+      identityref = true,
+      leafref = true,
```

Comment:
is the idea that leafrefs will be resolved later, somehow?  i.e. the parsed value is a name, and then later you have to look up that name